src/main/java/uz/bookstore/bookstore/dto/request/BookSaveDto.java

```diff
@@ -0,0 +1,39 @@
+package uz.bookstore.bookstore.dto.request;
+
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class BookSaveDto {
+
+    @NotNull
```

Comment:
Remove @NotNull , @NotEmpty e.t.c annotations. Because, we have Validations class and not null or empty method in it. User that method in services. We must write in one standard.